lib/SetAPI/genome/GenomeSetInterfaceV1.py

```diff
@@ -24,18 +34,24 @@ def save_genome_set(self, ctx, params):
             'set_info': info
         }
 
-    def _validate_genome_set_data(self, data):
+    def _validate_genome_set_data(self, data, save_search_set):
         # TODO: add checks that only one copy of each genome data is in the set
+        if save_search_set:
+            if 'elements' not in data:
+                raise ValueError('"elements" list must be defined in data to save an KBaseSearch.GenomeSet')
```

Comment:
... to save a KBaseSearch.GenomeSet, not 'an'

Reply:
And is the 'elements' part of the GenomeSet object documented anywhere? Could you add it as an option to the spec if not?